scalameta/dialects/shared/src/main/scala/scala/meta/Dialect.scala

```diff
@@ -150,7 +150,9 @@ final class Dialect private (
     // import a.b.c.{ given, _} used for -X:source3
     val allowGivenImports: Boolean,
     // Scala 3 uses proper precedence rules for infix types, unlike Scala 2
-    val useInfixTypePrecedence: Boolean
+    val useInfixTypePrecedence: Boolean,
+    // Scala213Source3 and Scala3 allow infix operator being placed after nl
```

Comment:
```suggestion
    //  Scala212Source3 , Scala213Source3 and Scala3 allow infix operator being placed after nl
```

Reply:
@tgodzik @dos65 in that case, shouldn't this new flag be added to scala212source3 dialect below?

Reply:
Ach, my bad. I looked at the dialects and thought it was added. Nevermind me then!